### ISSUE TITLE
Editable annotation points [perhaps redundant]

### ISSUE THREAD

**Issue by gpaar** [vrvis/PRo3D#215]
It shall be possible to pick points from annotations (line, polygon, D&S etc.) and modify / delete them or add additional points between two others or at the end

**Comment by gpaar** [comment]
See also https://github.com/pro3d-space/PRo3D/issues/205

**Comment by AndrewAnnex** [comment]
I would like to re-prioritize this issue to be more important now that the sample count improvements have been changed.

**Comment by haraldsteinlechner** [comment]
i agree. clearly there are requirements from different stakeholders (projects, users, research agendas), however i think it would be beneficial to have a tag/label for issues regarded as important in the community. We can discuss this in discord - i think prio:community-high or something would help a lot...
